### ISSUE TITLE
Non-steam applications with installer:  Cannot launch app after installation

### ISSUE THREAD

**Issue by AncientLich** [ValveSoftware/steam-for-linux#6518]
#### System information

* Steam client version: Built - Sept 20 2019 / Api v020 / Package version 1568941497
* Distribution (e.g. Ubuntu): Ubuntu 16.04 - 64bit
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

I use "add a non-steam game" (forcing Proton 3.16-9)  from steam client, and execute a windows installer (for example a game purchased on GOG or the Uplay client installer) the installer works fine and the game/application will be installed, as expected, in "$HOME/.steam/steam/steamapp/compatdata/<fake_game_id>/pfx" 

But when it is the time to "USE" the installed application, nothing works. 
1) I changed the "target" pointing to the file "$HOME/.steam/steam/steamapp/compatdata/<fake_game_id>/pfx/PATH/TO/EXE/application.exe" 
2) I changed the "start from" pointing to the directory "$HOME/.steam/steam/steamapp/compatdata/<fake_game_id>/pfx/PATH/TO/EXE/" 

But when hitting "play" NOTHING happens (while in wine at least the application started, even if works weird). If I opt-out "force proton 3.16" instead, for example for uplay, it complains that windows system is old (I assumed that Proton should set wine prefix as windows 7 at least, since steam support for windows xp and vista was ended)

#### Note: Bug for non-steam games in Proton 4 major releases

If I force Proton using one of the 4.*.* version, the INSTALLER will not be launched at all.... the compatdata directory is created BUT NOT populated. So I was forced to use Proton 3.16-9

**Comment by Myrddin-Wyllt** [comment]
This has happened to me & the issue was always the path wasn't properly set.
1) Under Target, put the exact case-sensitive name of the exe in "quotes".
2) Under Start In, put the exact case-sensitive path to the folder containing the above exe in "quotes".
3) Double check that Steam didn't throw garbage into the launch options (happens to me).

I've played a few non-Steam games via proton, but I never installed them with Proton. PlayOnLinux manual installs has been my method of choice; when a game doesn't run with Wine & I can't find a workaround, Proton has been able to run some of them. This is the only reason I've tried Proton for non-Steam games.

If you still have issues, this may be related to https://github.com/ValveSoftware/steam-for-linux/issues/6475

**Comment by AncientLich** [comment]
Thank a lot for your reply but there are NO errors on path... Under ubuntu it is possible to copy-paste the path of a file to clipboard and paste it with CTRL+V ... (replacing the file:///home with /home)... and I know that path must be quoted. So it is pretty impossible that the path is wrong. I will take a look to the other issue you linked

**Comment by NoXPhasma** [comment]
There is another bug which is also breaking stuff here. The moment I change anything to the shortcut, it resets the shortcuts category and removes the custom Proton version. Then I need to edit the shortcut again to enable Proton, which results in a new Prefix being created.

Which makes it needed to delete the new prefix, then rename the old one to the new one. Whatever they tried to fix with the [20th September beta update](https://steamcommunity.com/groups/SteamClientBeta#announcements/detail/1589131508768385391), it breaks the Proton behaviour.

**Comment by AncientLich** [comment]
Yeah true... when Proton is opted out (force proton unflagged) than Uplay (the application I am testing) complains that it is an old windows version (like the prefix was configured as windows xp or vista)... kinda weird since steam itself is supposed to not work in windows xp and vista so no reason why the compatdata prefix should be setted as xp... which can break some game configuration too... re-enabling "force to Proton 3.16" will again stop work. If I add the LDPATH suggested by the issue linked by Willt it seems it works the same way (complaining windows old version if the "force proton" is unflagged... not remember if it works if flagged...)

**Comment by kisak-valve** [comment]
Hello @AncientLich, are you still experiencing this issue on an up to date system?
